### DISCUSSION TITLE
Support for matplotlib animations

### DISCUSSION THREAD

**Discussion by SILIZ4** [quarto-dev/quarto-cli#6626] [Q&A] [answered]
### Description

I've started playing around with Quarto for over a week now and I love it coming from beamer. I'm wondering if there is any way to embed a matplotlib animation generated with [`funcanimation`](https://matplotlib.org/stable/api/_as_gen/matplotlib.animation.FuncAnimation.html). I don't know much about javascript so it might already be possible, but I couldn't find how.

I know that animations can be rendered in jupyter using "magic commands" such as `%matplotlib notebook` and using javascript with [`to_jshtml`](https://matplotlib.org/stable/api/_as_gen/matplotlib.animation.Animation.html#matplotlib.animation.Animation.to_jshtml).

Right now I'm saving the animation to a video and embedding in the video in the presentation. This is fine, but it would be much better if it was integrated in the .qmd document as when generating static figures.

Thank you for the help and this wonderful project!

**Comment by mcanouil** [comment, marked as answer]
The same thing should be possible in Quarto since it can reads directly from a Notebook.
Because this is mostly related to the library you are using rather than Quarto itself.
For further help, a reproducible document (or a simple Git repository) to work with is necessary.

You can share a Quarto document using the following syntax, _i.e._, using more backticks than you have in your document (usually four ` ```` `).

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
---

This is a reproducible Quarto document using `format: html`.
It is written in Markdown and contains embedded R code.
When you run the code, it will produce a plot.

```{r}
plot(cars)
```

The end.
````
`````

**Reply by SILIZ4** [reply]
Thank you for the quick reply. It was indeed a problem with my code, sorry for bothering.
